legacy/src/main/java/org/opensearch/sql/legacy/utils/StringUtils.java

```diff
@@ -90,12 +90,26 @@ public static String unquoteSingleField(String text) {
    * @return A string whose each dot-separated field has been unquoted from back-ticks (if any)
    */
   public static String unquoteFullColumn(String text, String quote) {
+    boolean startsWithPeriod = false;
+    if (text.startsWith(quote + ".")) {
+      startsWithPeriod = true;
+      text = quote + text.substring(2);
+    }
     String[] strs = text.split("\\.");
```

Comment:
Just wonder if any easier way to do this. Essentially, we just want to split from the second dot if text starts with `.` or `` `. `` ?